src/Bem.js

```diff
@@ -3,6 +3,28 @@ import stringify from '@bem/sdk.naming.entity.stringify';
 
 export default function({ preset, naming }) {
     const { Base, classAttribute, Render, PropTypes } = preset;
+    const getRenderProps = function(instance, node) {
+        const bemModes = {
+            block : 1,
+            elem : 1,
+            addBemClassName : 1,
+            tag : 1,
+            attrs : 1,
+            mods : 1,
+            mix : 1,
+            cls : 1
+        };
+
+        Object.assign(node, node.attrs, {
+            [classAttribute] : instance.__cnb(node)
+        });
+
+        return Object.keys(node).reduce((props, p) => {
```

Comment:
@veged we need to clean `props`, because react has a validation for undeclared `props` ;(